airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -191,7 +191,7 @@ public JsonNode toDatabaseConfig(final JsonNode config) {
     jdbcUrl.append("&tinyInt1isBit=true");
     // ensure the return year value is a Date; see the rationale
     // in the setJsonField method in MySqlSourceOperations.java
-    jdbcUrl.append("&yearIsDateType=true");
+    jdbcUrl.append("&yearIsDateType=false");
```

Comment:
The behavior I'm seeing doesn't match documentation. But I changed to `false` as it makes a YEAR to be read into a `SMALLINT` rather than a `DATE`.
But in both cases value converted correctly to the same integer.